index.html

```diff
@@ -0,0 +1,132 @@
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(2).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Tonic</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(3).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Multi-Post <br>Stories</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-myself">
+      <div class="primary-text">
+        <h1>About <br>Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+           of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social">
+        <div class="let-connect"><a>LET'S CONNECT</a></div>
+        <ul class="social-icons">
+          <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
+          <li><a href=""><img src="./images/linked.svg" alt=""></a></li>
+          <li><a href=""><img src="./images/medium.svg" alt=""></a></li>
+          <li><a href=""><img src="./images/github.svg" alt=""></a></li>
+          <li><a href=""><img src="./images/angellist.svg" alt=""></a></li>
+        </ul>
+      </div>
+      <div class="normal-button">
+        <button type="button">Get my resume</button>
+      </div>
+      <div>
+        <ul class="language-list">
+          <li><a href=""><img src="./images/Frame7.png" alt=""></a></li>
+          <li><a href=""><img src="./images/Frame9.png" alt=""></a></li>
+          <li><a href=""><img src="./images/Frame10.png" alt=""></a></li>
+          <li><a href=""><img src="./images/Frame11.png" alt=""></a></li>
+          <li><a href=""><img src="./images/Frame8.png" alt=""></a></li>
+          <li><a href=""><img src="./images/Frame6.png" alt=""></a></li>
```

Comment:
- [ ] I suggest you unwrap the list of languages from a tag because they are not supposed to redirect you somewhere. It is just the list of the languages you know.